src/FormNetworks/abstractformnetwork.jl

```diff
@@ -0,0 +1,36 @@
+abstract type AbstractFormNetwork{V} <: AbstractITensorNetwork{V} end
+
+#Needed for interface
+bra_vertex_map(f::AbstractFormNetwork) = not_implemented()
+ket_vertex_map(f::AbstractFormNetwork) = not_implemented()
+operator_vertex_map(f::AbstractFormNetwork) = not_implemented()
+dual_index_map(f::AbstractFormNetwork) = not_implemented()
+tensornetwork(f::AbstractFormNetwork) = not_implemented()
+copy(f::AbstractFormNetwork) = not_implemented()
+derivative_vertices(f::AbstractFormNetwork) = not_implemented()
+bra_ket_vertices(f::AbstractFormNetwork, state_vertices::Vector) = not_implemented()
+
+bra(f::AbstractFormNetwork) = induced_subgraph(f, collect(values(bra_vertex_map(f))))
+ket(f::AbstractFormNetwork) = induced_subgraph(f, collect(values(ket_vertex_map(f))))
+function operator(f::AbstractFormNetwork)
+  return induced_subgraph(f, collect(values(operator_vertex_map(f))))
+end
```

Comment:
I think these don't make sense any more with the new map design?

If we use the new vertex suffix design I suggest below, these could search the vertices for the vertex suffix.

Reply:
Also it may be good to design these around functions `bra_vertices(f::AbstractFormNetwork) = filter(v -> last(v) == bra_vertex_suffix(f), vertices(f))` and then define `bra_network(f::AbstractFormNetwork) = induced_subgraph(f, bra_vertices(f))`.

Reply:
Though I would have thought maybe this should return networks with the vertices mapped back to the original ones (i.e. with the suffix removed).

Reply:
Yeah you're right they don't make sense. The latest commit should have fixed that and reflect the new `suffix` logic.